### PR TITLE
bh - Setup seeds.rb to initialize 20 images on db:setup and add scrip…

### LINKED ISSUE #5: Image Seeds
#### As a developer I want at least 20 images to be created when I setup the database.

Acceptance criteria:
- [ ] After running `rake db:setup` at least 20 images are shown in the
  homepage.
- [ ] When a new app is deployed on heroku (e.g., every review app), that app
  has at least 20 images on its homepage.

Dependencies:
- Image Index

### PR DESCRIPTION
**Pull request by blakeherrington**
Closes #5

### CODE REVIEW

**Comment by blakeherrington** [comment]
Creating pull request so a review app gets generated... testing now to make sure things look good.

**Comment by blakeherrington** [comment]
Looks like db:setup ran correctly on deployment! This is ready for review.

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,29 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+image_urls = %w(
+  http://images.nationalgeographic.com/wpf/media-live/photos/000/945/cache/surin-elephants_94569_990x742.jpg
+  http://images.nationalgeographic.com/wpf/media-live/photos/000/943/cache/morskie-oko-scene_94323_990x742.jpg
+  http://images.nationalgeographic.com/wpf/media-live/photos/000/936/cache/denali-glacial-pools_93692_990x742.jpg
+  http://images.nationalgeographic.com/wpf/media-live/photos/000/939/cache/minori-rainbow_93919_990x742.jpg
+http://images.nationalgeographic.com/wpf/media-live/photos/000/945/cache/surin-elephants_94569_990x742.jpg
```
Comment (bboe) [inline db/seeds.rb]: The indentation here doesn't look consistent. It's a great idea to look at the "Files changed" tab on the pull request once you make it.

**Comment by bboe** [comment]
Looks great, just a minor whitespace issue. When you're confident that's fixed (you can use a amend commit `--amend`) go a head and merge it in. If you'd like me to look again, however, let me know.

File: db/seeds.rb (hunk 1 above)
Comment (blakeherrington) [inline db/seeds.rb]: Good tip! I'll do that in the future. These urls were wrapping in rubymine so I think that's why I didn't realize it.

**Pull request merged**